### PR TITLE
Fix rebuilds, missing file header, Update ServiceBus package

### PR DESCRIPTION
**Pull request by simonporter**
- Fix rebuilds with files set to copy always
- Add missing file header
- Update ServiceBus package version

### CODE REVIEW

**Comment by cgillum** [comment]
@simonporter does this PR still need to be merged?

**Comment by cgillum** [comment]
@simonporter ping

**Comment by simonporter** [comment]
> @simonporter ping

Updated, would like to merge if I can get a signoff

File: src/DurableTask.ServiceBus/Common/Abstraction/ServiceBusAbstraction.cs [hunk 1]
```diff
@@ -468,11 +481,6 @@ public ServiceBusConnection(string namespaceConnectionString)
         {
         }
 
-        public ServiceBusConnection(string namespaceConnectionString, TimeSpan operationTimeout, Microsoft.Azure.ServiceBus.RetryPolicy retryPolicy = null)
-            : base(namespaceConnectionString, operationTimeout, retryPolicy)
-        {
-        }
```
Comment (cgillum) [inline src/DurableTask.ServiceBus/Common/Abstraction/ServiceBusAbstraction.cs]: This is a public API and therefore a breaking change. Any concerns regarding this?

Comment (jviau) [reply src/DurableTask.ServiceBus/Common/Abstraction/ServiceBusAbstraction.cs:474]: Consider just marking it `[Obsolete]` as well. As done here #408

**Comment by cgillum** [comment]
Closing this old PR in favor of https://github.com/Azure/durabletask/pull/408.

**Pull request closed without merging**